docs/se/openapi.adoc

```diff
@@ -153,15 +117,31 @@ Set these config properties in one of the config sources your app uses so the
 Helidon config system will load them. Often developers use `application.yaml` at the
 top level of the application JAR.
 
-== Accessing the OpenAPI document
-Now your Helidon SE application will automatically respond to an additional endpoint --
- `/openapi` -- and it will return the OpenAPI document describing the endpoints
-in your application.
+== Examples
+
+Helidon SE provides a link:{helidon-github-tree-url}/examples/openapi[complete OpenAPI example]
+based on the SE quick-start sample app which includes a model reader and a filter.
```

Comment:
"quick-start" -> "Quickstart" as in other docs?

Reply:
Fixed